CMakeLists.txt

```diff
@@ -32,8 +36,6 @@ include_directories( "${CMAKE_CURRENT_SOURCE_DIR}/stellarsolver")
 include_directories( "${CMAKE_CURRENT_SOURCE_DIR}/stellarsolver/astrometry")
 include_directories( "${CMAKE_CURRENT_SOURCE_DIR}/stellarsolver/astrometry/include")
 include_directories( "${CMAKE_CURRENT_SOURCE_DIR}/stellarsolver/astrometry/include/astrometry")
-include_directories("/usr/local/include")
```

Comment:
are you sure these aren't needed?

Reply:
Yeah, these directories should not be included unless it's a MacOS specific requirement in which case they should be enclosed in an if statement.